src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -790,7 +790,7 @@ private void printArgumentParamUsage(final StringBuilder sb, final String name,
             numSpaces = ARGUMENT_COLUMN_WIDTH;
         }
         printSpaces(sb, numSpaces);
-        final String wrappedDescription = WordUtils.wrap(argumentDescription, DESCRIPTION_COLUMN_WIDTH);
+        final String wrappedDescription = Utils.wrapParagraph(argumentDescription, DESCRIPTION_COLUMN_WIDTH);
```

Comment:
I think you can remove the dangling imports of apache wordUtils now.

Reply:
done